talk/owt/sdk/base/peerconnectiondependencyfactory.cc

```diff
@@ -126,6 +128,13 @@ void PeerConnectionDependencyFactory::
     if (!GlobalConfiguration::GetCustomizedVideoDecoderEnabled()) {
       decoder_factory.reset(new MSDKVideoDecoderFactory());
     }
+  } else {
+    if (!encoded_frame_) {
+      encoder_factory = std::move(webrtc::CreateBuiltinVideoEncoderFactory());
+    }
+    if (!GlobalConfiguration::GetCustomizedVideoDecoderEnabled()) {
```

Comment:
Move to line 142?

Reply:
refactored the code to be more reasonable here.